pyctbgui/services/SlowADCs.py

```diff
@@ -10,13 +12,13 @@ def setup_ui(self):
         pass
 
     def connect_ui(self):
-        for i in range(8):
+        for i in range(Defines.numSlowAdc):
```

Comment:
Why dont you do the same way you did for signals and dacs.. I had liked that.. you could do "slowadc.count"?

Reply:
Okay I will do that